exec.ps1

```diff
@@ -1,2 +1,2 @@
-docker build --no-cache -t auth0-golang-kube-app .
-docker run -it --env-file .env -p 3000:3000 auth0-golang-kube-app
\ No newline at end of file
+docker image build --no-cache -t auth0-golang-kube-app .
+docker container run -it --rm --env-file ~/Documents/env.dev -p 3000:3000 auth0-golang-kube-app
```

Comment:
Assuming PowerShell script is for Windows users? Shouldn't this path use `\` instead of `/` and does `~` mean anything to Windows?

Reply:
@andyhd Yep, good catch.  Lazy copy/paste on my part.